Xamarin.Forms.Platform.Android/FastRenderers/AutomationPropertiesProvider.cs

```diff
@@ -55,6 +55,10 @@ internal static void SetAutomationId(AView control, VisualElement element, strin
 				defaultContentDescription = control.ContentDescription;
 
 			string value = ConcatenateNameAndHelpText(element);
+
+			if (control is IButtonLayoutRenderer button)
```

Comment:
I think we're going to have to revisit this later, since this is only fixing the issue for Buttons, and it now likely affecting other controls, too. This can work for now, though. Thanks!